### DISCUSSION TITLE
Best Practice:  testing lightdom critical css?

### DISCUSSION THREAD

**Discussion by zeroedin** [RedHat-UX/red-hat-design-system#294] [Q&A]
How do you approach testing critical CSS with your components?  

Currently, in order to test the component, the surrounding dev environment has to load, including the JS to dynamically include the component.   This wrapper makes it harder to view your component pre-upgrade.  Do you have tips or tricks you use to test lightdom CSS?

**Comment by bennypowers** [comment]
We need a vis.reg. test framework before we can seriously answer this question.

The little 😈 on my shoulder is telling me "you forgot to include the lightdom CSS" is a legit test failure condition.

**Reply by zeroedin** [reply]
> The little 😈 on my shoulder is telling me "you forgot to include the lightdom CSS" is a legit test failure condition.

💯
